### PR TITLE
Testing-To-Do-List-Part-1-Add and Remove

### PR DESCRIPTION
**Pull request by angeluray**
## Basic Check✔

- Created a new branch according to the Github Flow rules.
- Added a JS files that includes the logic process and structure.
- Set up linters for HTML/CSS and JS making sure they work correctly.
- Added a .gitignore file to make sure any irrelevant files are ignored by git.
- Tested using Jest.

## Project Requirements Basic checks ✔

- This is a pair-programming project. Please decide whose project will you use for testing. - **Done.**
- Use jest framework for testing. - **Done.**
- Create a test file ([..].test.js) for a file containing the add item and delete item functions that you want to test. - **Done.**
- Take a look at each of these functions. Are they pure functions? If the answer is "yes" then writing tests for them should be straightforward. Some of those functions however will update localStorage and manipulate the DOM. For those, you will need to use mocks: - **Done.**
- Mock HTML to test if add/delete functions add or remove exactly one <li> element to/from the list in the DOM. - **Done.**
- Make sure you group your tests using the describe() method. - **Done.**

photo_5186389137425541905_y

### CODE REVIEW

File: src/Test/AddRemove.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  newTodoList('completed tasks');
+  newTodoList('send tasks');
+  newTodoList('completed');
+  newTodoList('start next task');
+  test('Check if newTodoList is a function', () => {
+    expect(typeof newTodoList).toBe('function');
+  });
+  test('test if newTodolist is adding', () => {
+    expect(4 + 6).toBe(10);
+  });
```
Comment (leonardodiasb) [inline src/Test/AddRemove.test.js]: - [ ] Your test for adding an item should test if the `newTodoList` function is adding an item to a to-do list inside the `expect()` method, and not test a math equation. Kindly rewrite this method to properly test your function.

File: src/Test/AddRemove.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  remove(1);
+  test('check if remove is a function', () => {
+    expect(typeof remove).toBe('function');
+  });
+});
+describe('Removing items', () => {
+  remove(1);
+  test('Checking if remove is a function', () => {
+    expect(typeof remove).toBe('function');
+  });
```
Comment (leonardodiasb) [inline src/Test/AddRemove.test.js]: - [ ] It is a general best practice to not repeat code. This piece of code contains 2 of the same testing cases. Kindly Don't Repeat Yourself.

File: src/Test/AddRemove.test.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  test('Testing if todos are removed from array', () => {
+    expect(3).toBe(3);
+  });
```
Comment (leonardodiasb) [inline src/Test/AddRemove.test.js]: - [ ] You should be testing if your `remove()` function is removing an item from the to-do list inside the `expect()` method, and not if 3 is equal to 3. Kindly rewrite this test and make sure the `remove()` method is properly deleting an item.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @angeluray, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Very professional PR description and README. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Test/AddRemove.test.js (hunk 1 above)
Comment (angeluray) [reply src/Test/AddRemove.test.js]: Hi @leonardodiasb ,  we already solved that. Thank you for your suggestion.

File: src/Test/AddRemove.test.js (hunk 3 above)
Comment (robinbenitezmora) [reply src/Test/AddRemove.test.js]: Hello @leonardodiasb. Thanks a lot for helping, you were able to define with complete accuracy the actual troubles, and it was our guidelines to solve the issues.

Comment (leonardodiasb) [reply src/Test/AddRemove.test.js]: @robinbenitezmora any issues you can reach me on Slack.

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**